Program.cs

```diff
@@ -1,4 +1,16 @@
 ﻿using System;
-Console.WriteLine ("Hello, World!");
+Console.Write ("Enter a word or phrase:");
+string text = Console.ReadLine ();
+text = text.Replace (" ", "").ToLower ();
+char[] ch = text.ToCharArray ();
+Array.Reverse (ch);
+string rev = new string (ch);
+bool b = text.Equals (rev); 
+if (b == true) {

```

Comment:
`Console.WriteLine ("" + text + " is a Palindrome!");`
`Console.WriteLine (" " + text + " is not a Palindrome!");`

Why you are using empty spaces before 'text'?
if and else is a one line, you can write it like below,
`if (b) Console.WriteLine ();`
`else Console.WriteLine ();`

And write the output with the input string. because if we entered the input "Step on no pets", the output is 
"steponnopets is a Palindrome!".